Events/src/main/java/com/github/swcraftlyon/meetup/Event.java

```diff
@@ -1,4 +1,13 @@
 package com.github.swcraftlyon.meetup;
 
 public class Event {
+    private String title;
```

Comment:
There's no setter, so it's possible to make it immutable by adding the **final** keyword